<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_checker_zip">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Creative ZIP File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_checker_img">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Creative Image Files</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="result_txt_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>551</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_checker_resize">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Creative Auto Resize</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMoloco_Creative_Checker">
    <property name="title">
     <string>Moloco Creative Checker</string>
    </property>
   </widget>
   <addaction name="menuMoloco_Creative_Checker"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
